app/component/itinerary/ItineraryPage.js

```diff
@@ -1171,13 +1200,21 @@ export default function ItineraryPage(props, context) {
         walking={walkPlan?.edges?.length > 0}
         biking={bikePlan?.edges?.length > 0 || !!bikePublicPlan?.edges?.length}
         driving={
-          (settings.includeCarSuggestions && carPlan?.edges?.length > 0) ||
+          (settings.includeCarSuggestions &&
+            (carPlan?.edges?.length > 0 ||
+              (config.carBoardingModes?.FERRY !== undefined &&
+                carPublicPlan?.edges?.length))) ||
           !!parkRidePlan?.edges?.length
         }
       />
     );
   }
 
+  const showCarPublicPlan =
+    settings.includeCarSuggestions &&
+    config.carBoardingModes?.FERRY !== undefined &&
+    carPublicPlan.carPublicItineraryCount > 0;
+
   const showAltBar =
```

Comment:
carPublicPlan.carPublicItineraryCount > 0 is enough

Reply:
FERRY checking is now removed. The idea behind checking for FERRY is that only in the configs that it is defined in, the UI will show ferries. Now there isn't a config field for removing carpublic itineraries in case they are found but not wanted. I did not remove settings.includeCarSuggestions.

Reply:
Can there be car + public itineraries if includeCarSuggestions === false ?

The logic gets a bit complicated because car itinerary is fetched for co2 comparison although user does not wish to get car itineraries.